src/main/java/Wessy.java

```diff
@@ -0,0 +1,397 @@
+import Exceptions.CommandNotFoundException;
```

Comment:
Great job on importing the classes explicitly, and ordered them in a logical way.